### PR TITLE
Feature/update documentation

### PR DESCRIPTION
**Pull request by bryantson**
- Updated the documentation to make clearer on how to change TLS enabled value by adding the images and steps to follow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,26 @@ Once an successful installation of Che/CRW is verified, tests patch custom resou
 * enable oAuth
 * enable TLS mode
 
+You can check and turn ono/ff the TLS mod from ConfigMap by following the steps:
```
Comment (nickboldt) [inline README.md]: typo... on/off

File: README.md [hunk 2]
```diff
@@ -156,6 +156,26 @@ Once an successful installation of Che/CRW is verified, tests patch custom resou
...
+```sh
+oc describe configmap che
+```
+![oc edit configmap che](./images/TLSEnabledFlag_2.jpg)
+
+3. See the value for **CHE_INFRA_OPENSHIFT_TLS_ENABLED**
+
 Subsequent checks verify that the installation is reconfigured, for example uses secure routes or ConfigMap has the right Login-with-OpenShift values
 
 TODO: add more scenarios
```
Comment (nickboldt) [inline README.md]: Could you add more scenarios here? :)

File: README.md [hunk 3]
```diff
@@ -156,6 +156,26 @@ Once an successful installation of Che/CRW is verified, tests patch custom resou
...
+
+![oc get configmap](./images/TLSEnabledFlag_1.jpg)
+
+2. Describe or edit Configmap che to see the value **CHE_INFRA_OPENSHIFT_TLS_ENABLED**
+```sh
+oc edit configmap che
+```
+or
+```sh
+oc describe configmap che
```
Comment (nickboldt) [inline README.md]: what about:

```
oc describe configmap che| grep CHE_INFRA_OPENSHIFT_TLS_ENABLED
```

So you only see what you want to see? Then step 3 is no longer needed.

**Comment by tolusha** [comment]
Should be done in che-docs project

**Pull request closed without merging**